src/core_plugins/timelion/public/directives/interval/interval.html

```diff
@@ -1,8 +1,13 @@
-<input input-focus ng-show="interval === 'other'"
-  class="form-control timelion-interval"
-  ng-model="otherInterval">
+<input
+  input-focus
+  ng-show="interval === 'other'"
+  ng-class="{'form-control form-group': !inHeader, 'kuiLocalSearchInput kuiLocalSearchInput--secondary timelion-interval': inHeader}"
+  ng-model="otherInterval"
+>
 
-<select class="form-control timelion-interval timelion-interval--select"
+<select
+  class="timelion-interval"
   ng-options="i for i in intervalOptions"
```

Comment:
I think this is the syntax [ng-options](https://docs.angularjs.org/api/ng/directive/ngOptions) expects. Is there a different expression we should use?